hydrographic_core_msgs/msg/RawTextStamped.msg

```diff
@@ -0,0 +1,8 @@
+# message to contain raw ASCII data from a sensor.
+
+# this message should be timestamped at the receive time of the first character of the message.
+
+# the frame_id should be that of the sensor which sent the message
```

Comment:
I think we should explicitly say that frame_id is meaningless, but we use `std_msgs/Header` for timestamps for consistency with everything else in ROS.

For example, I could imaging using this RawData message to log communications with a device on the robot that doesn't actually show up in the TF tree...

Reply:
I think we should leave the option open:  
The frame_id should be that of the sensor which sent the message if a valid frame exists.  Otherwise this may be left blank.

Reply:
How about "For many use cases for this message, frame_id is meaningless and should be left blank. For sensor data, frame_id may be set to that of the sensor which sent the message."?